Mathlib/RepresentationTheory/Homological/GroupHomology/Functoriality.lean

```diff
@@ -448,6 +464,167 @@ previous assumptions. -/
 
 end OfTrivial
 
+/-- The short complex `H₁(S, A) ⟶ H₁(G, A) ⟶ H₁(G ⧸ S, A_S)`. -/
+@[simps X₁ X₂ X₃ f g]
+noncomputable def H1CoresCoinf :
```

Comment:
I realize now that I don't know what the "Coinf" in this name stands for... Would you mind adding that to the docstring? (And maybe some other (module) docstrings.)

Reply:
Thanks! Sorry for the slow reply, I've tried to add more docs